internal/selfupdate/github.go

```diff
@@ -129,13 +129,9 @@ func getGithubData(ctx context.Context, url string) ([]byte, error) {
 		return nil, fmt.Errorf("unexpected status %v (%v) returned", res.StatusCode, res.Status)
 	}
 
-	buf, err := ioutil.ReadAll(res.Body)
-	if err != nil {
-		_ = res.Body.Close()
-		return nil, err
-	}
+	defer res.Body.Close()
```

Comment:
This ignores errors closing the body. Not likely to happen, but no reason to take the risk either.